.gitignore

```diff
@@ -18,3 +18,4 @@ public/styleguide/
 autoload_classmap.php
 .mysql
 .vagrant/
+public/img/captcha/
```

Comment:
Well, uh, adding empty folders is not really something you should be doing with git. As the code will create a captcha, before looking for one, this does not lead to problems.